.github/project.yml

```diff
@@ -1,4 +1,4 @@
 release:
-  current-version: "0"
-  next-version: "1.0.0-SNAPSHOT"
+  current-version: "2.10.0.0"
```

Comment:
I don't have strong opinion for the version name. Is that okay to have the 4 digits ?
It seems other projects have the current-major-minor-version.  Maybe we can add the `current-major-minor-version: "2.10.0"

Reply:
Yeah I am relying on https://github.com/quarkiverse/quarkus-github-api, it is versionned `x.y.z` where `x.y` is the github-api upstream version and `z` is any evolution done on top of it if I am correct. @gsmet any thoughts ?

Reply:
Hi @gastaldi do you have any thoughts or suggestions on how we should version this? thanks!

Reply:
We can also consider versioning according to Quarkus major version 3.x.y. Again, I'm not certain. It would make sense to use the sshd version if it was released within the sshd project. Personally, I'm not in favor of using four digits. However, I understand that different approaches may have their merits, and I don't have a strong opinion on this matter.

Reply:
Hey there. Really depends on your needs. The GitHub API one is the only project for which I adopted this version scheme because it was really about pushing GitHub API versions as is and it moves fast enough that I wanted people to be aware of the version that was included.

It could have gotten problematic for the Quarkus 3 move though if the old versions were incompatible.

There's no bullet proof way of doing it unfortunately.

Reply:
Although @gsmet statement is correct, I'd recommend sticking to `x.y.z` as that's the semver format we tested in the registry. The proposed format may work but I am not sure we have tests covering this scheme.

Reply:
Thanks to both of you !!! 🙏🏼 
@zbendhiba let's make the first version `1.0.0`, sounds good to you ?

Reply:
Sounds good!